WordPress/Classes/Models/Blog.m

```diff
@@ -691,7 +691,8 @@ - (BOOL)supportsPluginManagement
 - (BOOL)supportsStories
 {
     BOOL hasRequiredJetpack = [self hasRequiredJetpackVersion:@"9.1"];
-    return hasRequiredJetpack || self.isHostedAtWPcom;
+    // Stories are disabled in iPad until this Kanvas issue is solved: https://github.com/tumblr/kanvas-ios/issues/104
+    return (hasRequiredJetpack || self.isHostedAtWPcom) && ![UIDevice isPad] && ![JetpackFeaturesRemovalCoordinator shouldRemoveJetpackFeatures];
```

Comment:
I noticed that we weren't using the same conditions for determining if the Story Post action should be shown in different places. For this reason, I decided to unify the logic into a single function. Specifically, checking if the device is an iPad. Based on [this PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/15996) and [this Kanvas issue](https://github.com/tumblr/kanvas-ios/issues/104), I understand that the Story post shouldn't be enabled on iPad.